packages/memo-intl/bench/bench.config.ts

```
@@ -0,0 +1,6 @@
+import isCi from 'is-ci';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import-x/namespace)